azext_iot/operations/dps.py

```diff
@@ -14,7 +14,7 @@
 from azext_iot.common.utility import shell_safe_json_parse
 from azext_iot.common.certops import open_certificate
 from azext_iot.operations.generic import _execute_query
-from azext_iot._factory import _bind_sdk
+from azext_iot._factory import SdkResolver
```

Comment:
nit - should this _factory be changed to factory since the _ implies private?

Reply:
Yea agreed there is a peculiarity with the underscore here, but will punt for later.